### PR TITLE
#162261075 Setup continuous Integration

### PR DESCRIPTION
**Pull request by kakaemma**
#### What does this PR do?
This PR adds continuous integration to the application ,

### How should this be manually tested?
The badges for build, test coverage and maintainability should be displayed in the README

#### What are the relevant pivotal tracker stories?
[#162261075](https://www.pivotaltracker.com/story/show/162261075)

#### Any background context you want to add?
N/A

### CODE REVIEW

File: src/tests/components/__snapshots__/Login.test.js.snap [hunk 1]
```diff
@@ -0,0 +1,170 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```
Comment (MuhanguziDavid) [inline src/tests/components/__snapshots__/Login.test.js.snap:1]: @kaka, please add snapshots files to your `.gitignore`. You could use this: `**/__snapshots__/**`

Comment (kakaemma) [reply src/tests/components/__snapshots__/Login.test.js.snap:1]: @MuhanguziDavid , according to the jest documentation, snapshot files are supposed to be included. Initially, I had them added to the `.gitignore` and the tests were failing.
[https://jestjs.io/docs/en/snapshot-testing](https://jestjs.io/docs/en/snapshot-testing)

**Review by hadijahkyampeire — APPROVED** [review]
Looks Good, just wondering why you are using two CIs

**Review by MuhanguziDavid — APPROVED** [review]
LGTM

**Comment by kakaemma** [comment]
@hadijahkyampeire , I have removed travis CI

**Pull request merged**